src/ProjectTemplates/test/ProjectExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Templates.Test.Helpers;
+
+namespace Templates.Test;
+
+public static class ProjectExtensions
+{
+    public static async Task<Project> VerifyLaunchSettings(this Project project, string[] expectedLaunchProfileNames)
+    {
+        using var launchSettingsFile = File.OpenRead(Path.Combine(project.TemplateOutputDir, "Properties", "launchSettings.json"));
```

Comment:
Is there be any benefit to using the config apis instead of json to verify things?  I guess since we don't own either now, we should use whichever is easier to read/maintain.

Reply:
You mean literally using `IConfiguration` to read and navigate the `launchSettings.json` file? It might be technically possible, but `launchSettings.json` doesn't feed into app configuration usually (that's `appsettings.json`) so it's possible it won't map super well to the configuration limitations. Verifying using JSON APIs was easy enough.